keycloak-openapi-transformer/src/paths.rs

```diff
@@ -9,8 +9,10 @@ mod verb_path;
 use verb_path::VerbPath;
 
 lazy_static! {
-    static ref TAG_SECTION_SELECTOR: Selector =
+    static ref TAG_SECTION_SELECTOR_DEPRECATED: Selector =
         Selector::parse("#_paths + .sectionbody > .sect2").unwrap();
+    static ref TAG_SECTION_SELECTOR: Selector =
+        Selector::parse("#_resources + .sectionbody > .sect2").unwrap();
```

Comment:
As this is CSS selectors, can we combine them with a comma? 

`"#_paths + .sectionbody > .sect2, #_resources + .sectionbody > .sect2"`

---

Thank you catching this issue and figuring out the solution. I wasn't even aware Keycloak 22.0.0 had released 😅

Reply:
Can do :)